lib/Doctrine/Connection.php

```diff
@@ -207,6 +207,11 @@
             'indexes' => array()
         );
 
+    /**
+     * @var Doctrine_Manager $parent   the parent of this component
+     */
+    protected $parent;
```

Comment:
This matches the visibility of `Doctrine_Configurable` which this class extends (so visibility isn't changed), just wanted to be able to document the type of the property to match what's used in this class.